src/backend/src/app.js

```diff
@@ -58,47 +52,27 @@ process.on('exit', (code) => {
  * @param {string} options.migrationsFolderPath
  * @param {string} options.sharedStoragePath Path to app-specific external file storage folder
  * @param {string} options.defaultConfigPath
- * @param {string} options.fallbackMapPath Path to app-specific external file storage folder
- *
  */
 export async function init({
   version,
   rootKey,
   migrationsFolderPath,
-  sharedStoragePath,
+  // sharedStoragePath,
   defaultConfigPath,
-  fallbackMapPath,
 }) {
   log('Starting app...')
   log(`Device version is ${version}`)
 
   const privateStorageDir = rnBridge.app.datadir()
   const dbDir = join(privateStorageDir, DB_DIR_NAME)
   const indexDir = join(privateStorageDir, CORE_STORAGE_DIR_NAME)
-  const staticStylesDir = join(sharedStoragePath, 'styles')
```

Comment:
the removal of this means that devices that have installed the app previously will have a lingering directory, which shouldn't be a problem but might be something worth documenting somehow?